### PR TITLE
update rules location

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/414

### CODE REVIEW

**Comment by e0i** [comment]
@VLG17 Please change the target branch of the PR from `live` to `master` to continue with the review.

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@JohanFreelancer9 Minor style checks needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by dariomws** [comment]
Mmmm be careful as I think it should be removed from the ConfigurableServiceStore bullet. And now I see there is more to put outside this bullet. I think the description of this parameter could be:

Targets the policy store from which to retrieve the rules to be retrieved. A policy store is a container for firewall and IPsec policy. The acceptable values for this parameter are:

- PersistentStore: Default value. Sometimes called static rules, this store contains the persistent policy for the local computer. This policy is not from GPOs, and has been created manually or programmatically (during application installation) on the computer. Rules created in this store are attached to the ActiveStore and activated on the computer immediately.
- ActiveStore: This store contains the currently active policy, which is the sum of all policy stores that apply to the computer. This is the resultant set of policy (RSOP) for the local computer (the sum of all GPOs that apply to the computer), and the local stores (the PersistentStore, the static Windows service hardening (WSH), and the configurable WSH). ---- GPOs are also policy stores. Computer GPOs can be specified as follows. ------ -PolicyStore hostname. ---- Active Directory GPOs can be specified as follows. ------ -PolicyStore domain.fqdn.com\GPO_Friendly_Namedomain.fqdn.comGPO_Friendly_Name. ------ Such as the following. -------- -PolicyStore localhost -------- -PolicyStore corp.contoso.com\FirewallPolicy ---- Active Directory GPOs can be created using the New-GPO cmdlet or the Group Policy Management Console.
- RSOP: This read-only store contains the sum of all GPOs applied to the local computer.
- SystemDefaults: This read-only store contains the default state of firewall rules that ship with Windows Server 2012.
- StaticServiceStore: This read-only store contains all the service restrictions that ship with Windows Server 2012. Optional and product-dependent features are considered part of Windows Server 2012 for the purposes of WFAS.
- ConfigurableServiceStore: This read-write store contains all the service restrictions that are added for third-party services. 

In addition, network isolation rules that are created for Windows Store application containers are stored in the registry under HKLM:\SYSTEM\CurrentControlSet\Services\SharedAccess\Parameters\FirewallPolicy\RestrictedServices\AppIso\FirewallRules and aren't accessible with Get-NetFirewallRule. 
The Set-NetFirewallRule cmdlet cannot be used to add an object to a policy store. An object can only be added to a policy store at creation time with the Copy-NetFirewallRule or with the New-NetFirewallRule cmdlet.

**Comment by dariomws** [comment]
Bold, greyed text, etc must be added.

**Comment by VLG17** [comment]
> * PersistentStore: Default value. Sometimes called static rules, this store contains the persistent policy for the local computer.

@dariomws Thank you for providing this. I did not think to check the Get-NetFirewallSetting article for details about this. I'll add it, but this part sound a bit off to me:

"targets the policy store from which to **retrieve the rules to be retrieved**" 

What do you think would be the best wording here?

**Comment by dariomws** [comment]
@VLG17 yes, that's true too.

Maybe:
"Targets the policy store from which to retrieve the rules"
Is enough.

@get-itips what do you think?

**Comment by get-itips** [comment]
> @VLG17 yes, that's true too.
> 
> Maybe:
> "Targets the policy store from which to retrieve the rules"
> Is enough.
> 
> @get-itips what do you think?

Yes

**Comment by e0i** [comment]
@tseward Good to go if you don't have any additional comments. Thanks.

**Comment by tseward** [comment]
@e0i good to go on my side.

File: docset/windows/netsecurity/Get-NetFirewallRule.md [hunk 1]
```diff
@@ -648,10 +648,7 @@ Computer GPOs can be specified as follows.
...
 Optional and product-dependent features are considered part of Windows Server 2012 for the purposes of WFAS. 
 - ConfigurableServiceStore: This read-write store contains all the service restrictions that are added for third-party services.
-In addition, network isolation rules that are created for Windows Store application containers will appear in this policy store. 
-The default value is PersistentStore. 
-The Set-NetFirewallRule cmdlet cannot be used to add an object to a policy store.
-An object can only be added to a policy store at creation time with the Copy-NetFirewallRule or with the New-NetFirewallRule cmdlet.
+In addition, network isolation rules that are created for Windows Store application containers are stored in the registry under **HKLM:\SYSTEM\CurrentControlSet\Services\SharedAccess\Parameters\FirewallPolicy\RestrictedServices\AppIso\FirewallRules** and aren't accessible with **Get-NetFirewallRule**.The default value is PersistentStore. The **Set-NetFirewallRule** cmdlet cannot be used to add an object to a policy store.An object can only be added to a policy store at creation time with the **Copy-NetFirewallRule** or with the **New-NetFirewallRule** cmdlet.
```
Comment (JohanFreelancer9) [inline docset/windows/netsecurity/Get-NetFirewallRule.md]:
```suggestion
In addition, network isolation rules that are created for Windows Store application containers are stored in the registry under **HKLM:\SYSTEM\CurrentControlSet\Services\SharedAccess\Parameters\FirewallPolicy\RestrictedServices\AppIso\FirewallRules** and aren't accessible with **Get-NetFirewallRule**.The default value is PersistentStore. The **Set-NetFirewallRule** cmdlet cannot be used to add an object to a policy store. An object can only be added to a policy store at creation time with the **Copy-NetFirewallRule** or with the **New-NetFirewallRule** cmdlet. 
```

**Pull request merged**